.tekton/kueue-fbc-418-push.yaml

```diff
@@ -0,0 +1,397 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift/kueue-operator?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "false"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "push" && target_branch
+      == "main"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: kueue-fbc-418
+    appstudio.openshift.io/component: kueue-fbc-418
+    pipelines.appstudio.openshift.io/type: build
+  name: kueue-fbc-418-on-push
+  namespace: kueue-operator-tenant
+spec:
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image
+    value: quay.io/redhat-user-workloads/kueue-operator-tenant/kueue-fbc-418:{{revision}}
+  - name: build-platforms
+    value:
+    - linux/amd64
+  - name: dockerfile
+    value: /fbc/v4.18/Containerfile.catalog
```

Comment:
I think you need:

  - name: path-context 
    value: fbc/v4.18 
    
    Here also.

Reply:
Yes.  Thanks